CMakeLists.txt

```diff
@@ -81,6 +81,22 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(ktl)
 
+# dr_libs for dr_wav
+FetchContent_Declare(
```

Comment:
Should we make this a git submodule instead? They show up on GitHub, auto update commit hashes, etc; more ergonomic except requires DAG, which is fine here since the dependency is private.

Reply:
Yes, I thought you were going for using FetchContent for every dependency